<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBPrintTemplateListDialog</class>
 <widget class="QDialog" name="RBPrintTemplateListDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список записей</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="3">
    <widget class="QLineEdit" name="edtCodeFilter"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lblNameFilter">
     <property name="text">
      <string>Наименование</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="edtNameFilter"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblCodeFilter">
     <property name="text">
      <string>Код</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtContextFilter"/>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="CSortFilterProxyTableView" name="tblItems">
     <property name="whatsThis">
      <string comment="ура!">список записей</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblContextFilter">
     <property name="text">
      <string>Контекст</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QStatusBar" name="statusBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>A status bar</string>
     </property>
     <property name="whatsThis">
      <string>A status bar.</string>
     </property>
     <property name="sizeGripEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>всего: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSortFilterProxyTableView</class>
   <extends>QTableView</extends>
   <header>library/SortFilterProxyTableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tblItems</tabstop>
  <tabstop>edtCodeFilter</tabstop>
  <tabstop>edtNameFilter</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RBPrintTemplateListDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
